api/api/src/main/scala/uk/ac/wellcome/platform/api/models/SearchTemplates.scala

```diff
@@ -0,0 +1,22 @@
+package uk.ac.wellcome.platform.api.models
+
+import com.sksamuel.elastic4s.requests.searches.queries.{Query, QueryBuilderFn}
+import io.circe.generic.extras.semiauto.deriveConfiguredEncoder
+import io.circe.Encoder
+import uk.ac.wellcome.display.json.DisplayJsonUtil._
+
+case class SearchTemplates(templates: List[SearchTemplate])
+object SearchTemplates {
+  implicit def encoder: Encoder[SearchTemplates] =
+    deriveConfiguredEncoder
+}
+
+case class SearchTemplate(id: String, index: String, query: String)
+
+object SearchTemplate {
```

Comment:
minor: I’m not a huge fan of variables/classes whose names differ by a single character.

Reply:
I agree with this. Also is `SearchTemplates` necessary, why not just use `List[SearchTemplate]` directly?

Reply:
As I would then need to add it to the top level JSON object:
`{ "templates": [] }`

Most of this is for JSON rendering, so I don't see a massive problem with it.

Reply:
Ah I get it. How about calling it `SearchTemplatesResponse` to indicate it's the top level response that is returned at that endpoint?

Reply:
Yip, that makes sense.